clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -432,8 +438,8 @@ public void testSerialization() throws Exception {
         checkRequest(createUpdateMetadataRequest(5, null), false);
         checkErrorResponse(createUpdateMetadataRequest(5, "rack1"), unknownServerException, true);
         checkResponse(createUpdateMetadataResponse(), 0, true);
-        checkRequest(createListOffsetRequest(0), true);
-        checkErrorResponse(createListOffsetRequest(0), unknownServerException, true);
+        checkRequest(createListOffsetRequest(0, 1000000L), true);
+        checkErrorResponse(createListOffsetRequest(0, 1000000L), unknownServerException, true);
```

Comment:
Is this redundant with what is already tested above?

Reply:
yep, removed.